great_expectations/rule_based_profiler/attributed_resolved_metrics.py

```diff
@@ -148,11 +148,14 @@ def add_resolved_metric(self, batch_id: str, value: MetricValue) -> None:
 
     @property
     def id(self) -> str:
+        if self.metric_attributes is None:
+            return ""
+
         return self.metric_attributes.to_id()
```

Comment:
```suggestion
        return self.metric_attributes.to_id() or ""
```

Reply:
@Kilo59 Suggestion applied -- thanks.

Reply:
In the end, I had to revert on the account of static type check warnings.